### DISCUSSION TITLE
Average Price paid calculation and display

### DISCUSSION THREAD

**Discussion by d1964b** [BottlecapDave/HomeAssistant-OctopusEnergy#303] [General]
Is there an easy way to use yesterdays consumption and cost figures to calculate the average price paid in p/kWh and then display that value on a card ? 
After a frustrating period of trial and error I finally managed to calculate the figure , which just displays as a number ( in Dev Tools \ Template) using this : 

{{ (states('sensor.octopus_energy_electricity_xxxxx_yyyyy_previous_accumulative_cost') | float(0) * 100 ) / states('sensor.octopus_energy_electricity_xxxxx_yyyyy_previous_accumulative_consumption') | float(0) }}

but I don't know how to assign that value to anything so I can display it.

**Comment by BottlecapDave** [comment]
You will need to use HA's [template](https://www.home-assistant.io/integrations/template/) feature to create a sensor based on your calculations.
